src/js/tabs/osd.js

```diff
@@ -306,6 +306,22 @@ OSD.generateAltitudePreview = function(osdData) {
     return `${FONT.symbol(SYM.ALTITUDE)}399${variantSelected === 0? '.7' : ''}${unit}`;
 };
 
+
+OSD.generateVTXChannelPreview = function(osdData) {
+    const variantSelected = OSD.getVariantForPreview(osdData, 'VTX_CHANNEL');
+    let value;
+    switch (variantSelected) {
+        case 0:
+            value = `R:2:200:P`;
```

Comment:
You dont need to use ` you can use ' or " if it isn't a template.
```suggestion
            value = 'R:2:200:P';
```